facebook_connector.py

```diff
@@ -10,15 +10,15 @@
 from __future__ import print_function
 
 from bs4 import BeautifulSoup
-from fbchat_archive_parser.parser import MessageHtmlParser
+from fbchat_archive_parser.parser import parse
```

Comment:
When there's a risk of conflict it's better to write:

from fbchat_archive_parser import parser then below parser.parse()